metals/src/main/scala/scala/meta/internal/tvp/MetalsTreeViewProvider.scala

```diff
@@ -267,7 +268,10 @@ class MetalsTreeViewProvider(
   ): Option[TreeViewNodeRevealResult] = {
     val input = path.toInput
     val occurrences =
-      Mtags.allToplevels(input).occurrences.filterNot(_.symbol.isPackage)
+      Mtags
+        .allToplevels(input, dialects.Scala213)
```

Comment:
Hardcoded 213 here. TreeViewProvider doesn't works with scala3 - #2859

Reply:
Could you add a comment above referencing the issue?